internal/gcsx/client_readers/range_reader.go

```diff
@@ -15,18 +15,25 @@
 package gcsx
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"io"
 
+	"github.com/googlecloudplatform/gcsfuse/v2/common"
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/fs/gcsfuse_errors"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/gcsx"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/logger"
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/gcs"
+	"github.com/googlecloudplatform/gcsfuse/v2/internal/util"
 )
 
 type RangeReader struct {
-	// TODO: Add additional fields as needed.
 	gcsx.Reader
-	start int64
-	limit int64
+	object *gcs.MinObject
+	bucket gcs.Bucket
+	start  int64
```

Comment:
Can we provide the brief description of start, limit? What is it?

Reply:
I think it is self-explanatory, but since you felt we should add it, so did I